### ISSUE TITLE
Make kubernetes controller ract faster to down nodes

### ISSUE THREAD

**Issue by etlweather** [rancher/rancher#14712]
Rancher 2.0.6

I covered this in details on the forum - https://forums.rancher.com/t/health-check-does-it-even-work/11062 - but wasn't able to get an answer.

I'm trying to make Kubernetes react to a down node faster. Right now, it takes 5 minutes before new pods are created to replace those that were running on the node that went down. This isn't acceptable for my scenario.

From the Kubernetes documentation, I found that I can change `node-monitor-period`, `node-monitor-grace-period` and `pod-eviction-timeout` to reduce this delay.

I tried putting this in the `cluster.yaml` config when creating the cluster but the configuration went away, except for `node-status-update-frequency` which remained but I'm not sure it made a difference.

```
  kublet:
    extra_args:
      node-status-update-frequency: "5s"
  kube-controller:
    extra_args:
      node-monitor-period: "2s"
      node-monitor-grace-period: "16s"
      pod-eviction-timeout: "30s"
```

What is the correct way to set those options? Is it even possible at this time?

**Comment by superseb** [comment]
I ran tests on this last week using:

```
  kubelet:
    extra_args:
      node-status-update-frequency: 4s
  kube_controller:
    extra_args:
      node-monitor-period: 2s
      node-monitor-grace-period: 16s
      pod-eviction-timeout: 30s
```

Can you share the exact `cluster.yml` used? As `node-status-update-frequency` is the one that stays, but the service definition has a typo `kublet`.

**Comment by etlweather** [comment]
Bingo!

Thank you for putting your config that worked there! I found the mistake. 

`kublet` was a typo when I transcribed it (it's an air gap system). But `kube-controller` is what I had entered and you had `kube_controller`.

The underscore was it.

Now I did put a dash because that's what I saw in RKE documentation. I don't know if it is wrong in RKE docs or if there is a difference how it handles things when through Rancher 2.0?

See https://rancher.com/docs/rke/v0.1.x/en/example-yamls/ - that uses `kube-controller`.
